utils/scan_parameters.py

```diff
@@ -188,6 +212,20 @@ def RunProjs(projs,hlsdir,kerasdir):
                 subprocess.call("rm -rf %s" % autopilot,
                             stdout = outlog, stderr=errlog, shell=True)
 
+            subprocess.call('mkdir %s/keras-to-hls/results_%s'%(hlsdir,k),
+                            stdout = outlog, stderr=errlog, shell=True)      
+         
+            if os.path.exists(output_filename):
```

Comment:
How the output_filename works here? It is pointed to the results_%s. Can't find where you copy it from the original project directory.

Reply:
You are right! This is an error which I forgot from some quick tests I was doing. I am fixing it.